src/gt4py/backend/gtc_backend/defir_to_gtir.py

```diff
@@ -197,6 +197,10 @@ def visit_BinOpExpr(self, node: BinOpExpr) -> Union[gtir.BinaryOp, gtir.NativeFu
             return gtir.NativeFuncCall(
                 func=common.NativeFunction.POW, args=[self.visit(node.lhs), self.visit(node.rhs)]
             )
+        if node.op == BinaryOperator.MOD:
+            return gtir.NativeFuncCall(
+                func=common.NativeFunction.MOD, args=[self.visit(node.lhs), self.visit(node.rhs)]
+            )
```

Comment:
An optional alternative way to write this to remove the almost duplicated `if`s for `POW` and `MOD`

```suggestion
        if node.op in (BinaryOperator.POW, BinaryOperator.MOD):
            return gtir.NativeFuncCall(
                func=common.NativeFunction[node.op.name], args=[self.visit(node.lhs), self.visit(node.rhs)]
            )
```

Reply:
Done.